docs/arguments.md

```diff
@@ -142,6 +142,39 @@ prefix these flags with `distributor.ha-tracker.`
 - `etcd.max-retries`
    The maximum number of retries to do for failed ops.
 
+#### memberlist
+
+Flags for configuring KV store based on memberlist library.
```

Comment:
Should there be a callout at the start of this section regarding this feature (& its flags) being experimental?

Reply:
Added EXPERIMENTAL warning here.